src/Admin/FieldDescriptionRegistryInterface.php

```diff
@@ -34,7 +34,7 @@ interface FieldDescriptionRegistryInterface
      *
      * @param string $name
      *
-     * @return FieldDescriptionInterface
+     * @return FieldDescriptionInterface|null
```

Comment:
This will be non nullable on master (just leaving a comment to remember on merge)

Reply:
I've added a comment with `NEXT_MAJOR`